### DISCUSSION TITLE
Team chat/communication feature

### DISCUSSION THREAD

**Discussion by lakkyaaxsync** [makeplane/plane#1196] [Q&A]
Is there a team communication feature in the roadmap as an alternative to Slack? For example, mattermost has integrated team chat feature and zulip has a fantastic threaded chat feature. any such addition would be game changing.

**Comment by pablohashescobar** [comment]
Hey there, @lakkyaaxsync! I'd love to hear your thoughts. To get started, I suggest creating a feature request. Once you've done that, we'll be sure to prioritize and address it as soon as possible.

**Reply by putt1ck** [reply]
Probably better off integrating Matrix rather than yet another chat system - that way people can integrate with whatever chat system they already have, whether corporate or personal.

**Comment by andresmascl** [comment]
Hi there.  I think having an integration with MS Teams would be awesome.  Not sure if I should open a feature request or this fits here, but I'll be happy to open one if you guys think it would be best

**Reply by putt1ck** [reply]
Teams should be covered by a Matrix integration, which would require a bridge - either using EMS or getting an open source bridge completed. Seems better effort than a direct Teams integration.

**Comment by robin-liquidium** [comment]
An integration for Mattermost would be very appreciated. I tried using the webhooks with mattermost and it doesn't seem to work. No events are sent to my channel in Mattermost.

Would be great if it was possible to receive direct notifications if I am mentioned in some issue or if someone comments on an issue that's assigned to me or that I'm following.
